examples/express_app.js

```diff
@@ -1,20 +1,28 @@
 const express = require('express');
-const { InteractionType, InteractionResponseType, verifyKeyMiddleware } = require('../dist');
+const {
+	InteractionType,
+	InteractionResponseType,
+	verifyKeyMiddleware,
+} = require('../dist');
 
 const app = express();
 
-app.post('/interactions', verifyKeyMiddleware(process.env.CLIENT_PUBLIC_KEY), (req, res) => {
-  const interaction = req.body;
-  if (interaction.type === InteractionType.APPLICATION_COMMAND) {
-    res.send({
-      type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
-      data: {
-        content: 'Hello world',
-      },
-    });
-  }
-});
+app.post(
+	'/interactions',
+	verifyKeyMiddleware(process.env.CLIENT_PUBLIC_KEY),
+	(req, res) => {
+		const interaction = req.body;
+		if (interaction.type === InteractionType.APPLICATION_COMMAND) {
+			res.send({
+				type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+				data: {
+					content: 'Hello world',
+				},
+			});
+		}
+	},
+);
```

Comment:
I'm not a fan of this tab spacing, can you make this match discord_app of 2?

Reply:
So this reformatted the entire codebase in tabs instead of spaces?




nice

Reply:
HUGE